packages/storage-mixin/lib/backends/secure-main.js

```diff
@@ -12,7 +13,12 @@ if (process && process.type === 'browser') {
   ipc.respondTo('storage-mixin:clear', (evt, meta) => {
     debug('Clearing all secure values for', meta.serviceName);
 
-    return (require('keytar').findCredentials(meta.serviceName))
+    return Promise.resolve()
+      .then(function() { return require('keytar').findCredentials(meta.serviceName); })
```

Comment:
Not sure if this can throw (as anticipated in `secure.js`) or only reject (as anticipated here), using `Promise.resolve()` puts this on the safe side and hopefully all of this will be async functions soon enough anyway.